specification/app/data-plane/Microsoft.App.DynamicSessions/preview/2024-10-02-preview/DynamicSessions.json

```diff
@@ -0,0 +1,759 @@
+        },
+        "status": {
+          "$ref": "#/definitions/CodeExecutionStatus",
+          "description": "The status of the code execution, indicates whether succeeded or not."
+        },
+        "stdout": {
+          "type": "string",
+          "description": "The standard output of the code execution."
+        },
+        "stderr": {
+          "type": "string",
+          "description": "The standard error of the code execution."
+        },
+        "result": {
+          "description": "The result of the code execution. The type of this field is same as the type of actual result of the code execution after being Json serialized."
+        },
+        "executionTimeInMilliseconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The execution time of the code in milliseconds."
+        }
+      },
+      "required": [
+        "id",
+        "status",
+        "stdout",
+        "stderr",
+        "result",
+        "executionTimeInMilliseconds"
+      ]
+    },
+    "SessionCodeExecutionRequest": {
+      "type": "object",
+      "description": "The request to execute code.",
+      "properties": {
+        "codeInputType": {
+          "$ref": "#/definitions/CodeInputType",
+          "description": "The code input type of the code execution request."
+        },
+        "executionType": {
+          "$ref": "#/definitions/ExecutionType",
+          "description": "The execution type of the code execution request."
+        },
+        "code": {
+          "type": "string",
+          "description": "The string of the code to execute, based on CodeInputType."
+        },
+        "timeoutInSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Code execution timeout in seconds."
+        }
+      },
+      "required": [
+        "codeInputType",
+        "executionType",
+        "code",
+        "timeoutInSeconds"
+      ]
+    },
+    "SessionCodeExecutionResponse": {
+      "type": "object",
+      "description": "The response containing the result of the code execution.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Session code execution id.",
+          "readOnly": true
+        },
+        "status": {
+          "$ref": "#/definitions/CodeExecutionStatus",
+          "description": "The status of the code execution, indicates whether succeeded or not."
+        },
+        "stdout": {
+          "type": "string",
+          "description": "The standard output of the code execution."
+        },
+        "stderr": {
+          "type": "string",
+          "description": "The standard error of the code execution."
+        },
+        "result": {
+          "description": "The result of the code execution. The type of this field is same as the type of actual result of the code execution after being Json serialized."
+        },
+        "executionTimeInMilliseconds": {
```

Comment:
What is the value of this field if the code execution has not finished? Is it the execution time so far? This should be explained in the description. Or make it optional and only return it when status is a terminal value.

Reply:
This would return null if the code execution has not finished.